pkgs/development/libraries/cxxtest/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, python, pythonPackages}:
+
+stdenv.mkDerivation rec {
+  version = "4.4";
+  name = "cxxtest";
+
+  src = fetchFromGitHub {
+    owner = "CxxTest";
+    repo = name;
+    rev = version;
+    sha256 = "19w92kipfhp5wvs47l0qpibn3x49sbmvkk91yxw6nwk6fafcdl17";
+  };
+
+  buildInputs = with pythonPackages; [ python wrapPython ];
+
+  installPhase = ''
+    cd python
+    python setup.py install --prefix=$out
```

Comment:
Oops, sorry.

Indeed, using `buildPythonPackage` I can remove the line 
```
wrapPythonProgramsIn $out/bin "$out $pythonPath"
```
but I have to add the lines
```
  buildPhase = ''
    pythonInstallDir=$out/lib/${pythonPackages.python.libPrefix}/site-packages
    mkdir -p $pythonInstallDir
    export PYTHONPATH=''${PYTHONPATH:+''${PYTHONPATH}:}$pythonInstallDir
    cd python
    python setup.py build
    cd ..
  '';
  doCheck = false;
```
so the first version seems simpler to me.